boudicca.base/publisher-event-html/src/main/kotlin/base/boudicca/publisher/event/html/PublisherHtmlConfiguration.kt

```diff
@@ -22,18 +26,22 @@ import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
 @AutoConfiguration
 class PublisherHtmlConfiguration(private val properties: PublisherHtmlProperties) : WebMvcConfigurer {
     @Bean
+    @Primary
```

Comment:
does primary really do something here?

Reply:
yeah, was a dead end i was going down when i thought the viewResolver.order was not working :) -> removed